app/views/candidate_mailer/chase_reference.text.erb

```diff
@@ -1,9 +1,27 @@
-<%= @reference.name %> has not responded yet.
+<% if @application_form.show_new_reference_flow? %>
```

Comment:
These are essentially two templates bound by 1 conditional in all cases in this PR.

I think it would be better to use partials to separate the two. When the new references flow becomes the only game in town we can then promote the relevant partial back to the mailer template.